File: 110algoritmos/exercicios018/ex018-TypeScript/ex018.ts
let angulo = Number(prompt("Digite um angulo: "))
let seno = Math.sin((angulo * Math.PI)/180.0)
let coseno = Math.cos((angulo * Math.PI)/180.0)
let tangente = Math.tan((angulo * Math.PI)/180.0)

console.log(`O SENO de ${angulo} é ${seno.toFixed(2)}`)
console.log(`O CONSENO de ${angulo} é ${coseno.toFixed(2)}`)
console.log(`O TANGENTE de ${angulo} é ${tangente.toFixed(2)}`)

File: 110algoritmos/exercicios018/ex018-Go/ex018.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var angulo float64

	fmt.Println("Digite o valor do angulo: ")
	fmt.Scanln(&angulo)

	seno := math.Sin((angulo * math.Pi) / 180.0)
	conseno := math.Cos((angulo * math.Pi) / 180.0)
	tangente := math.Tan((angulo * math.Pi) / 180.0)

	fmt.Printf("\nO SENO de %v é: %.2f", angulo, seno)
	fmt.Printf("\nO COSENO de %v é: %.2f", angulo, conseno)
	fmt.Printf("\nA TANGENTE de %v é: %.2f", angulo, tangente)

}
